### ISSUE TITLE
java.lang.IllegalStateException URI error

### ISSUE THREAD

**Issue by ssy248** [eclipse/deeplearning4j#8234]
I get the following error:

Exception in thread "main" java.lang.IllegalStateException: Attempting to open URI that is not a File URI; for other stream types, you must use an appropriate stream loader function. URI: C:/Users/super/Desktop/serenastraining/csvharrypotter/amrepwinglev/harrypotterfeb0.csv
	at org.nd4j.base.Preconditions.throwStateEx(Preconditions.java:641)
	at org.nd4j.base.Preconditions.checkState(Preconditions.java:304)
	at org.datavec.api.split.streams.FileStreamCreatorFunction.apply(FileStreamCreatorFunction.java:29)
	at org.datavec.api.split.streams.FileStreamCreatorFunction.apply(FileStreamCreatorFunction.java:25)
	at org.datavec.api.records.reader.impl.csv.CSVSequenceRecordReader.nextSequence(CSVSequenceRecordReader.java:79)
	at org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator.initializeUnderlyingFromReader(SequenceRecordReaderDataSetIterator.java:167)
	at org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator.hasNext(SequenceRecordReaderDataSetIterator.java:337)
	at org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator.next(SequenceRecordReaderDataSetIterator.java:358)
	at org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator.next(SequenceRecordReaderDataSetIterator.java:344)
	at org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator.preLoad(SequenceRecordReaderDataSetIterator.java:394)
	at org.deeplearning4j.datasets.datavec.SequenceRecordReaderDataSetIterator.inputColumns(SequenceRecordReaderDataSetIterator.java:381)
	at javaspark.deeplearning2.main(deeplearning2.java:88)

when I run my code: 

`int miniBatchSize = 10;		
int numPossibleLabels = 3;
		// use regression = false for classification
		boolean regression = false;
		
		SequenceRecordReader featureReader = new CSVSequenceRecordReader(1, ",");
		SequenceRecordReader labelReader = new CSVSequenceRecordReader(1, ",");
		
		featureReader.initialize(new NumberedFileInputSplit(datafiledir+"/harrypotterfeb%d.csv", 0, 22));
		
		labelReader.initialize(new NumberedFileInputSplit(labelfiledir+"/harrypotterfeblabel%d.csv", 0, 22));
		
		DataSetIterator iter = new SequenceRecordReaderDataSetIterator(featureReader, labelReader, miniBatchSize, numPossibleLabels, regression);

		
		// DataSetIterator can then be passed to MultiLayerNetwork.fit() to train the network.

		int seed = 100;
		
		// learning rate: hyperparameter that controls how much adjust weights of network w.r.t loss gradient
		// large value 0.1, and then lower 0.01 and 0.001, etc
		double learningRate=0.1;
		
		// https://github.com/deeplearning4j/dl4j-examples/blob/master/dl4j-examples/src/main/java/org/deeplearning4j/examples/recurrent/character/LSTMCharModellingExample.java
		// LSTM parameters
		int lstmLayerSize = 200;	
		int nOut= 3;
        int tbpttLength = 50;                       //Length for truncated backpropagation through time. i.e., do parameter updates ever 50 characters

 MultiLayerConfiguration conf = new NeuralNetConfiguration.Builder()
				.seed(10)
				.l2(0.0001)
	            .weightInit(WeightInit.XAVIER)
	            .updater(new Adam(0.005))
				.list()
				.layer(new LSTM.Builder().nIn(iter.inputColumns()).nOut(lstmLayerSize)
						.activation(Activation.TANH).build())
				.layer(new LSTM.Builder().nIn(lstmLayerSize).nOut(lstmLayerSize)
						.activation(Activation.TANH).build())
				//.layer(new RnnOutputLayer.Builder(LossFunctions.MCXENT).activation(Activation.SOFTMAX)        //MCXENT + softmax for classification
				//.nIn(lstmLayerSize).nOut(nOut).build())
	            .backpropType(BackpropType.TruncatedBPTT).tBPTTForwardLength(tbpttLength).tBPTTBackwardLength(tbpttLength)
				.build();`

**Comment by ssy248** [comment]
Ok, thanks. I tried updating the version of datavec to 1.0.0-beta5 but it gives the same error..

**Comment by raphaelvilela** [comment]
The **SequenceRecordReader**  checks the schema of file, so you need to set it. Just use the file:/// . See:

`NumberedFileInputSplit inputFiles = new NumberedFileInputSplit("file:///C:/inputs-%d.csv", 0, 1);`
